fhir-config/pom.xml

```diff
@@ -20,6 +20,16 @@
             <groupId>org.glassfish</groupId>
             <artifactId>jakarta.json</artifactId>
         </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.8.0</version>
```

Comment:
please remove the version and put this in dependencyManagement (same with commons-text)

Reply:
good call.  done.